platform/os_windows.go

```diff
@@ -189,6 +208,188 @@ func SetSdnRemoteArpMacAddress() error {
 	return nil
 }
 
+func HasMellanoxAdapater() bool {
```

Comment:
```suggestion
func HasMellanoxAdapter() bool {
```

Reply:
Made changes to code to correct this typo